src/main/java/org/jitsi/jigasi/transcription/GoogleCloudTranscriptionService.java

```diff
@@ -894,7 +895,7 @@ void terminateCurrentSession()
             this.languageTag = languageTag;
             this.debugName = debugName;
 
-            messageID = UUID.randomUUID();
+            messageID = Generators.timeBasedGenerator().generate();
```

Comment:
Can you make `messageID` final please?

Reply:
There's another instance of `UUID.randomUUID()` on line 358. Maybe update at least for consistency?

Reply:
Done.